src/connection/StreamSocket.php

```diff
@@ -5,6 +5,8 @@
 
 use Bolt\Bolt;
 use Bolt\error\ConnectException;
+use function floor;
```

Comment:
if you are not aliasing base php function then there is no reason to add "use"

Reply:
https://veewee.github.io/blog/optimizing-php-performance-by-fq-function-calls/
It is a small micro optimization so I will remove the `use` statement but there is a valid case to do this

Reply:
That can be true but after all it means, we should add "use" for every function?
My question is, if we load php file without "use function" and somewhere in the code we call some function, the second call of this function will be already optimized because the name resolve was made before? This "use function" has meaning only for repeatly loaded php file?

Reply:
To be honest I am not completely sure :man_shrugging:  It is something my IDE complains about and does automatically. I think it is on a file per file basis. I removed the statement now to stay consistent with the library.

Reply:
Now I'm curious. :D Thanks for keeping the consistency.

Reply:
No problem! I am quite sure it is on a file by file basis from reading the article. EA php inspections in phpstorm has this warning